src/woo_publications/publications/api/serializer.py

```diff
@@ -29,14 +29,38 @@ class Meta:  # type: ignore
         )
 
 
+class EigenaarSerializer(serializers.Serializer):
+    display_name = serializers.CharField(
+        read_only=True,
+        help_text=_(
+            "obtained from the audit trail request headers, extracted from the audit trails/log entries."
+        ),
+    )
+    identifier = serializers.CharField(
+        read_only=True,
+        help_text=_(
+            "arbitrary, unique-ish string. Could be a `sub` claim from OIDC, but for our perspective this data has no 'meaning'."
+        ),
```

Comment:
the comments in the ticket were meant as a hint for you, the developer, not as the help text of the field :stuck_out_tongue: 

```suggestion
        help_text=_("The system identifier that uniquely identifies the user performing the action."),
```
text taken from `src/woo_publications/api/drf_spectacular/headers.py`